cmd/schedule-builder/cmd/root.go

```diff
@@ -45,17 +45,20 @@ type options struct {
 	configPath string
 	outputFile string
 	logLevel   string
+	typeFile   string
 }
 
 var opts = &options{}
 
 const (
 	configPathFlag = "config-path"
 	outputFileFlag = "output-file"
+	typeFileFlag   = "type-file"
```

Comment:
```suggestion
	typeFlag   = "type"
```

Reply:
Thank you so much, I will fix this :smiley: